crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -444,27 +436,17 @@ pub mod wasm_test {
         // the chain.
         let countersign_fail_create_alice = conductor
             .raw_handle()
-            .call_zome(
-                ZomeCall::try_from_params(
-                    conductor.raw_handle().keystore(),
-                    ZomeCallParams {
-                        cell_id: alice.cell_id().clone(),
-                        zome_name: alice.name().clone(),
-                        fn_name: "create_an_invalid_countersigned_thing".into(),
-                        cap_secret: None,
-                        provenance: alice_pubkey.clone(),
-                        payload: ExternIO::encode(vec![
-                            alice_response.clone(),
-                            bob_response.clone(),
-                        ])
-                        .unwrap(),
-                        nonce,
-                        expires_at,
-                    },
-                )
-                .await
-                .unwrap(),
-            )
+            .call_zome(ZomeCallParams {
```

Comment:
Nice, so we no longer need to set up signing for direct calls in sweettest, that seems like it's now how it should always have been :D

Reply:
Exactly. It felt unnerving when removing all this "security" in the code, until it sank in that it's missing the point to authenticate internal calls.